src/controller/CHIPDeviceController.cpp

```diff
@@ -928,6 +930,9 @@ CHIP_ERROR DeviceCommissioner::Commission(NodeId remoteDeviceId, CommissioningPa
         ChipLogError(Controller, "Invalid device for commissioning " ChipLogFormatX64, ChipLogValueX64(remoteDeviceId));
         return CHIP_ERROR_INCORRECT_STATE;
     }
+
+    mCommissioningStage = kSecurePairing;
```

Comment:
Why this change?

Reply:
Stale change from way back to fix a bug in this state machine that has since been fixed.